dialogue-core/src/main/metrics/dialogue-core-metrics.yml

```diff
@@ -9,6 +9,10 @@ namespaces:
         type: timer
         tags: [service-name]
         docs: Request time, note that this does not include time spent reading the response body.
+      requests.active:
+        type: counter
+        tags: [stage]
+        docs: Number of requests that are actively running. The stage may refer to 'in-flight' requests actively executing over the wire, or 'processing' which may be awaiting a client or backing off for a retry.
```

Comment:
Looking for feedback on naming `dialogue.client.requests.active` should probably use singular `request`.
stage `in-flight`: maybe rename to `wire`.
`processing` is fairly vague, but as such it's not going to be incorrect.

Reply:
`request.active` seems alright.  Agree `in-flight` is a bit confusing since from a users perspective the request is `in-flight` as soon as it hits our code.  Perhaps `running` and `queued` for consistency with CJR?

Reply:
I like it, thanks!

Reply:
oh hmm, queued isn't quite right because it includes everything, both queued and running.